tests/engine-state/detect-engine-state-03/input.t

```diff
@@ -0,0 +1,15 @@
+>>>
+POST /upload.cgi HTTP/1.1
+Host: www.server.lan
+Content-Type: multipart/form-data; boundary=---------------------------277531038314945
+Content-Length: 215
+

```

Comment:
This looks the same as for detect-engine-state-02, is that correct?

If that's the case, then test 3 could just pass the pcap from test 2 as argument, instead of us having duplicated pcaps. (see https://github.com/OISF/suricata-verify/blob/master/tests/tls/tls-subject/test.yaml#L1 for an example on how to do that :) )

Reply:
No! test 3 makes use of two buffers, test 2 makes use of one, that's why they are in different test folders.

Reply:
Ok, right, I checked everything else but that aspect.